pkg/logql/bench/bench_test.go

```diff
@@ -150,8 +150,11 @@ func BenchmarkLogQL(b *testing.B) {
 
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					_, err := q.Exec(ctx)
+					r, err := q.Exec(ctx)
 					require.NoError(b, err)
+					b.ReportMetric(float64(r.Statistics.TotalDecompressedLines()), "linesScanned")
+					b.ReportMetric(float64(r.Statistics.TotalChunksDownloaded()), "chunks/dataobjSections")
+					b.ReportMetric(float64(r.Statistics.Summary.TotalPostFilterLines), "postFilterLines")
```

Comment:
aside about why I retracted my review for temporarily: 

I was double checking we didn't need to divide by `b.N` here; since we're doing it in the loop it'll just keep the most recent result, which I think in our case is fine 👍